### DISCUSSION TITLE
No Module Requests

### DISCUSSION THREAD

**Discussion by ZoozleDev** [bckd00r/LViewPlus#27] [Q&A]
![lview](https://user-images.githubusercontent.com/85362882/122423892-2e0b6c80-cf86-11eb-91a5-ce416c90a04c.png)

![Screenshot_3](https://user-images.githubusercontent.com/85362882/122423960-3cf21f00-cf86-11eb-846a-103308ecc7fd.png)

I know that this has been answered by saying pip install requests

but thats simply not the case, I have had no problems running either LVIEW the original or Billy's variation but when it comes to running this one, it says that requests doesn't exist when its installed....

**Comment by bckd00r** [comment]
You have to do it by entering python 3.9

**Comment by bckd00r** [comment]
![image](https://user-images.githubusercontent.com/48347237/122509697-dfed7c00-d00c-11eb-916a-1deffa50b788.png)

**Comment by ZoozleDev** [comment]
Okay, so I did that and now it crashes when it loads "ashews"
![Screenshot_4](https://user-images.githubusercontent.com/85362882/122542603-cbb47980-d022-11eb-8cf3-f39ca0ddc9f7.png)
